rocksdb/tests/test_db.py

```diff
@@ -489,6 +490,44 @@ def test_prefix_iteritems(self):
         ret = takewhile(lambda item: item[0].startswith(b'00002'), it)
         self.assertEqual(ref, dict(ret))
 
+class TestFixedPrefixExtractor(TestHelper):
+    def setUp(self):
+        TestHelper.setUp(self)
+        opts = rocksdb.Options(create_if_missing=True, prefix_extractor=4)
+        self.db = rocksdb.DB(os.path.join(self.db_loc, 'test'), opts)
+
+    def _fill_db(self):
+        for x in range(3000):
+            keyx = hex(x)[2:].zfill(5).encode('utf8') + b'.x'
```

Comment:
I know this is not your code but FWIW this line can be written as

```python
keyx = b'%05x.%b' % (x, b'x')
```

(Python 3.6+)

Reply:
Will do